piccolo/utils/lazy_loader.py

```diff
@@ -0,0 +1,60 @@
+# https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/util/lazy_loader.py
+from __future__ import absolute_import, division, print_function
+
+import importlib
+import types
+import typing as t
+
+
+class LazyLoader(types.ModuleType):
+    """
+    Lazily import a module.
+
+    `PostgresEngine` and `SQLiteEngine` are example use cases.
+    """
+
+    # The lint error here is incorrect.
+    def __init__(
+        self, local_name, parent_module_globals, name
+    ):  # pylint: disable=super-on-old-class
+        self._local_name = local_name
+        self._parent_module_globals = parent_module_globals
+
+        super(LazyLoader, self).__init__(name)
```

Comment:
@aminalaee Do we have to use the old style `super` here, or can we just use `super().__init__(name)`, and drop the `pylint: disable=super-on-old-class` comment?

Reply:
@dantownsend You're right. That works. I just go that from the link in code I didn't give it much thought. Thanks